Makefile

```diff
@@ -52,8 +52,11 @@ js:  ## build javascript
 labextension: js ## enable labextension
 	jupyter labextension install .
 
-dist: clean  ## dist to pypi
-	python3 setup.py sdist bdist_wheel && twine upload -r pypi dist/*
+dist: js  ## dist to pypi
+	rm -rf dist build
+	python3 setup.py sdist
+	python3 setup.py bdist_wheel
```

Comment:
Note that this runs the JS build step twice. Combining sdist and wheel build is faster, but not strictly necessary.